indico/MaKaC/webinterface/rh/conferenceDisplay.py

```diff
@@ -538,25 +544,29 @@ def _process( self ):
 
 
 class RHAuthorIndex(RHConferenceBaseDisplay):
-    _uh=urlHandlers.UHConfAuthorIndex
+    _uh = urlHandlers.UHConfAuthorIndex
+    _MENU_LINK_TO_DISABLE = 'authorIndex'
 
-    def _checkParams( self, params ):
-        RHConferenceBaseDisplay._checkParams( self, params )
+    def _checkParams(self, params):
+        RHConferenceBaseDisplay._checkParams(self, params)
 
     def _process(self):
-        p=conferences.WPAuthorIndex(self,self._target)
+        p = conferences.WPAuthorIndex(self,self._target)
         return p.display()
 
+
 class RHSpeakerIndex(RHConferenceBaseDisplay):
-    _uh=urlHandlers.UHConfSpeakerIndex
+    _uh = urlHandlers.UHConfSpeakerIndex
+    _MENU_LINK_TO_DISABLE = 'speakerIndex'
 
     def _checkParams( self, params ):
         RHConferenceBaseDisplay._checkParams( self, params )
 
     def _process(self):
-        p=conferences.WPSpeakerIndex(self,self._target)
+        p = conferences.WPSpeakerIndex(self,self._target)
```

Comment:
there's still a pep8 violation in this line. but i wouldn't bother cleaning up MaKaC code you don't actually modify. it'll be replaced later this year anyway...